x-pack/plugins/observability_solution/dataset_quality/server/services/data_telemetry/constants.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DATA_DATASETS_INDEX_PATTERNS_UNIQUE } from '@kbn/telemetry-plugin/server/telemetry_collection/get_data_telemetry/constants';
+
+import { DatasetIndexPattern } from './types';
+
+export const LOGS_DATA_TELEMETRY_TASK_TYPE = 'logs-data-telemetry';
+export const LOGS_DATA_TELEMETRY_TASK_ID = 'logs-data-telemetry:collect-and-report-task-2';
+
+export const TELEMETRY_TASK_INTERVAL = 24 * 60; // 24 hours (in minutes)
+export const TELEMETRY_TASK_TIMEOUT = 18 * 60; // 18 hours (in minutes)
```

Comment:
It can take 18 hours for this task to run?  Hoping that's a mistake.  Most tasks use 1-2m, some up to 5, a very few up to like 30m.  An 18h timeout would be the new record :-)

Reply:
18 hours was an initial guess, to have the task run in the background in a very lazy manner, but yes it's too high of a number.

The implementation is changed anyway, so now the value is 10 minutes.